flexinput/src/main/java/com/lytefast/flexinput/fragment/PhotosFragment.kt

```diff
@@ -49,7 +50,13 @@ open class PhotosFragment : PermissionsFragment() {
     return view?.apply {
       recyclerView = findViewById(R.id.list)
 
-      val photoAdapter = PhotoCursorAdapter(context.contentResolver, selectionCoordinator!!)
+      val display = activity?.windowManager?.defaultDisplay
+      val size = Point()
+      display?.getSize(size)
+
+      val thumbnailWidth = (size.x / 3f).toInt()
+
+      val photoAdapter = PhotoCursorAdapter(context.contentResolver, selectionCoordinator!!, thumbnailWidth, thumbnailWidth * 2)
```

Comment:
can we comment why we're multiplying by 2?

Reply:
will add some info here